tools/asn_parser_generator.py

```diff
@@ -0,0 +1,613 @@
+import sys
```

Comment:
Move the file to `scripts` or move the `settings` scripts here too.

Reply:
Hum if you don't mind I'll move the `settings` scripts in `tools` as I find the name more appropriate

Reply:
ok, just don´t want to have scripts split over too many different folders meaning more or less the same